core/src/main/java/org/apache/carbondata/core/metadata/PartitionMapFileStore.java

```
@@ -206,10 +206,10 @@ private PartitionMapper readPartitionMap(String partitionMapPath) {
           Charset.forName(CarbonCommonConstants.DEFAULT_CHARSET));
       buffReader = new BufferedReader(inStream);
       partitionMapper = gsonObjectToRead.fromJson(buffReader, PartitionMapper.class);
-    } catch (IOException e) {
-      return null;
```

Here is the modification, should not ignore the exception